control_v2.proto

```diff
@@ -213,7 +213,7 @@ message UpgradeDetails {
 message UpgradeDetailsMetadata {
   // If the upgrade is a scheduled upgrade, the timestamp of when the
   // upgrade is expected to start.
-  google.protobuf.Timestamp scheduled_at = 1;
+  string scheduled_at = 1;
```

Comment:
This change is not related to this PR, but I'm introducing another time field in this same struct in this PR so it was convenient to make their types the same.  String is the correct type here; I realized that after I noticed how `VersionResponse.buildTime` was typed.

Reply:
just to double check, it hasn't been released yet, therefore it isn't a breaking change, right?

Reply:
Correct.